matching-notification-api/src/main/resources/ApplicationResources.properties

```diff
@@ -181,3 +180,9 @@ phenotips.myMatches.contact.preview.error.header=Error getting contact email pre
 phenotips.myMatches.notification.dialog.success.title=Success
 phenotips.myMatches.notification.dialog.error.title=Error
 phenotips.myMatches.notification.dialog.closeButton.name=Close
+
+phenotips.myMatches.disclaimer.matchesLastRun=Matches with this database were last computed on
+phenotips.myMatches.disclaimer.noMatchRequest=Matches with this database were never computed.
+phenotips.myMatches.disclaimer.showDisclaimer=Show disclaimer
+phenotips.myMatches.disclaimer.disclaimerTitle=Service disclaimer:
+phenotips.myMatches.disclaimer.disclaimer.PhenomeCentral=<p>PhenomeCentral provides data in good faith as a research tool, but without verifying the accuracy, clinical validity or utility of the data. PhenomeCentral makes no warranty, express or implied, nor assumes any legal liability or responsibility for any purpose for which the data are used.</p><br><p>The user certifies that no attempt to identify individual patients will be undertaken.</p><br><p>Prior to using PhenomeCentral data in a publication, the user will contact the owner of the matching dataset to assess the integrity of the match and if validated will offer appropriate agreed recognition of their contribution, which may include co-authorship if the magnitude of the contribution warrants it to at least one representative from the project/participating centre (possibly the member who submitted the patient data).</p><br><p>If PhenomeCentral enables a research publication, the authors must acknowledge PhenomeCentral using the following wording: &ldquo;This study makes use of data shared through the PhenomeCentral repository. Funding for PhenomeCentral was provided by Genome Canada and Canadian Institute of Health Research (CIHR).&rdquo;</p>
```

Comment:
It's a bad practice to put markup in translation strings. I would have opted for a different approach:
- [ ] Place the disclaimer in its own wiki document, at `PhenomeCentral/ServiceDisclaimer`, with the title "PhenomeCentral Service Disclaimer" and following content (this is wiki markup):
```
PhenomeCentral provides data in good faith as a research tool, but without verifying the accuracy, clinical validity or utility of the data. PhenomeCentral makes no warranty, express or implied, nor assumes any legal liability or responsibility for any purpose for which the data are used.

The user certifies that no attempt to identify individual patients will be undertaken.

Prior to using PhenomeCentral data in a publication, the user will contact the owner of the matching dataset to assess the integrity of the match and if validated will offer appropriate agreed recognition of their contribution, which may include co-authorship if the magnitude of the contribution warrants it to at least one representative from the project/participating centre (possibly the member who submitted the patient data).

If PhenomeCentral enables a research publication, the authors must acknowledge PhenomeCentral using the following wording: "//This study makes use of data shared through the PhenomeCentral repository. Funding for PhenomeCentral was provided by Genome Canada and Canadian Institute of Health Research (CIHR).//"
```
- [ ] Load the content of the document like this:
```#set ($disclaimer = "$!xwiki.getDocument('PhenomeCentral.ServiceDisclaimer').getRenderedContent()")```

The advantage of having the disclaimer in its own document is that we can link it and display it as is for the users on the PC website.